### PR TITLE
Periodic publish

### PR DESCRIPTION
**Pull request by muniter**
The idea of this pull request is to improve periodic publishing to include lights, and also send the current state of light and sensor when connect/reconnect. This addresses #22 

Changes:
1. Refactor publishing of sensors and lights to a method each.
    - _publish_sensor
    - _publish_light

    Allowing to reuse in other parts of the code.

2. Refactor periodic publish to also publish lights state without
   repeating code. Implementing logic to send in the given tele period.

3. Addition on _connect_forever sending of sensor and light states after
   connecting. Therefore keeping the state in sync on reconnections.

If you agree with the changes I can update the readme to reflect this new configuration option. I'm willing to make the changes you consider appropriate in the code.

### CODE REVIEW

File: lumimqtt/__main__.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def main():
         'sensor_threshold': 50,  # 5% of illuminance sensor
         'sensor_debounce_period': 60,  # 1 minute
         'light_transition_period': 1.0,  # second
+        'light_tele_period': 60, # 1 minute
```
Comment (devbis) [inline lumimqtt/__main__.py:54]: What does `tele` mean in this case?

Comment (muniter) [reply lumimqtt/__main__.py:54]: I'm not a native English speaker. But I've seen in multiple projects and contexts the use of 'tele' short word for [telemetry](https://www.britannica.com/technology/telemetry). I've seen it in tasmota and other contexts. That's why I named it like that.

But I could change it to something like frequency if you like it better.

Comment (devbis) [reply lumimqtt/__main__.py:54]: This is not telemetry as I can see. What about changing it to a more clear `light_notification_period`?

Comment (muniter) [reply lumimqtt/__main__.py:54]: Sure thing will change it right away.

Comment (muniter) [reply lumimqtt/__main__.py]: Changes have been made 😄

File: lumimqtt/lumimqtt.py [hunk 2]
```diff
@@ -301,23 +299,47 @@ async def _periodic_publish(self, period=1):
...
+                aio_mqtt.ConnectionClosedError,
+                aio_mqtt.ServerDiedError,
+            ) as e:
+                logger.error("Connection closed", exc_info=e)
+                await self._client.wait_for_connect()
+                continue
+
             await aio.sleep(period)
 
+    async def _publish_sensor(self, sensor: Sensor, value=False):
```
Comment (devbis) [inline lumimqtt/lumimqtt.py:323]: Change the default value to None, please. It is not a boolean value. 
Also, zero value should be allowed.

File: lumimqtt/lumimqtt.py [hunk 3]
```diff
@@ -301,23 +299,47 @@ async def _periodic_publish(self, period=1):
...
+        value = value or sensor.get_value()
```
Comment (devbis) [inline lumimqtt/lumimqtt.py:324]: Here also must be replaced to explicit check for None. 0 is a valid sensor value.

Comment (muniter) [reply lumimqtt/lumimqtt.py]: Oh right, thank you! just fixed.

**Comment by devbis** [comment]
Thanks!

**Pull request merged**